### PR TITLE
{tools}[GCC/4.9.2] parallel 20150822 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/52365fd3b41fce7a5f9f for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.22.0-GCC-4.9.2-bare.eb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = 'Perl'
+version = '5.22.0'
+versionsuffix = '-bare'
+
+homepage = 'http://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Perl/Perl-5.22.0-GCC-4.9.2-bare.eb]: 4.9.3?

File: easybuild/easyconfigs/p/parallel/parallel-20150822-GCC-4.9.2.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+name = 'parallel'
+version = '20150822'
+
+homepage = 'http://savannah.gnu.org/projects/parallel/'
+description = """parallel: Build and execute shell commands in parallel"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://ftp.gnu.org/gnu/parallel']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/parallel/parallel-20150822-GCC-4.9.2.eb]: `GNU_SOURCE`?

File: easybuild/easyconfigs/p/Perl/Perl-5.22.0-GCC-4.9.2-bare.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.22.0-GCC-4.9.2-bare.eb]: I need this to be compatible with `intel/2015a`, hence 4.9.2

File: easybuild/easyconfigs/p/parallel/parallel-20150822-GCC-4.9.2.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/p/parallel/parallel-20150822-GCC-4.9.2.eb]: nice catch, will fix

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e136a11b215cc6495db3 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**